documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1421,7 +1421,9 @@ The string representation of the interval value that follows the pattern `P<year
 |`BYTES` or `STRING`
 |n/a +
  +
-Either the raw bytes (the default), a base64-encoded string, or a hex-encoded string, based on the connector's {link-prefix}:{link-postgresql-connector}#postgresql-property-binary-handling-mode[binary handling mode] setting.
+Either the raw bytes (the default), a base64-encoded string, or a hex-encoded string, based on the connector's {link-prefix}:{link-postgresql-connector}#postgresql-property-binary-handling-mode[binary handling mode] setting. +
+ +
+Debezium only supports Postgres `bytea_output` configuration of value `hex`.
```

Comment:
Could you add a link to the PG docs discussing this option?

Reply:
Updated!